server/model/monitor.js

```diff
@@ -26,12 +26,22 @@ const { UptimeCacheList } = require("../uptime-cache-list");
  */
 class Monitor extends BeanModel {
 
+    statusToKey(status) {
+        switch (status) {
+            case 0: return "down";
+            case 1: return "up";
+            case 2: return "pending";
+            case 4: return "maintenance";
+            default: return "unknown";
+        }
+    }
+
     /**
      * Return an object that ready to parse to JSON for public
      * Only show necessary data to public
```

Comment:
```suggestion
     * Only show necessary data to public
     * @param {boolean} [includeStatus = false] Should the JSON include the status
```

Also, it might be a good idea to change this to `showStatus` to match the `toPublicJSON` method in `server/model/group.js`